### PR TITLE
fix(ui): Updated skipped pairing warnings check

### PR DESCRIPTION
**Pull request by amanCypherock**

### CODE REVIEW

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-6529 On device in onboarding the user is skipping card auth if disconnected after pairing just one card](https://app.clickup.com/t/9002019994/PRF-6529)

File: src/restricted_app/restricted_host_interface.c [hunk 1]
```diff
@@ -115,7 +115,6 @@ void restricted_host_interface(engine_ctx_t *ctx,
   /* Device authentication is complete, reset the flow as the core will now need
    * to render the main menu */
   if (DEVICE_AUTHENTICATED == get_auth_state()) {
-    delay_scr_init(ui_text_check_cysync_app, DELAY_TIME);
```
Comment (ujjwal-cyph) [inline src/restricted_app/restricted_host_interface.c:118]: is this intentional? link any request for change to this.

**Review by ujjwal-cyph — APPROVED** [review]
LGTM

File: src/restricted_app/restricted_host_interface.c (hunk 1 above)
Comment (amanCypherock) [reply src/restricted_app/restricted_host_interface.c:118]: Yes, once we complete the authentication process, this screen is displayed, which seems unnecessary.

Comment (ujjwal-cyph) [reply src/restricted_app/restricted_host_interface.c:118]: yes, that makes sense. does not seem as an issue.

just needed to double check.

**Pull request merged**